maui/src/Accordion/AccordionItem.cs

```diff
@@ -462,11 +462,6 @@ static void OnIsExpandedPropertyChanged(BindableObject bindable, object oldValue
 			// Content does not get collapsed when item is being collapsed in PCL view.
 			if (bindable is AccordionItem accordionItem)
 			{
-				if (accordionItem.Content != null && (bool)newValue && accordionItem.Content.IsVisible)
-				{
-					accordionItem.Content.IsVisible = true;
-				}
-
```

Comment:
I struggle understanding why we are removing this 🙂

Reply:
@bricefriha - This part of the code was not relevant to the current issue and has been removed, as it was unnecessary.